src/Dfe.PlanTech.Web/Views/Shared/ChangeSchool.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@model string
+
+<div class="school-ident">
+		<div class="school-ident-content">
+		<h2 class="govuk-heading-s school-ident-name">@Model</h2>
+		<a class="govuk-link" href="/groups/select-a-school">Change school</a>
+	</div>
+</div>
```

Comment:
It looks like this uses `string` as a model instead of the newly-created `ChangeSchoolViewModel`.

Looking at the usages it might be easiest to remove the model and keep it as a string, but I don't have any preference in one direction or the other.

Also a nitpick re: some inconsistent indentation here.